File: Golang/2455-average-value-of-even-numbers-that-are-divisible-by-three/average-value-of-even-numbers-that-are-divisible-by-three.go
func averageValue(nums []int) int {
    sum, count := 0, 0
    for _, num := range nums {
        if num % 2 == 0 && num % 3 == 0 {
            sum += num
            count++
        }
    }
    if count == 0 {
        return 0
    }
    return int(sum / count)
}

File: Golang/2542-average-value-of-even-numbers-that-are-divisible-by-three/average-value-of-even-numbers-that-are-divisible-by-three.py
class Solution:
    def averageValue(self, nums: List[int]) -> int:
        sum = count = 0
        for num in nums:
            if num % 2 == 0 and num % 3 == 0:
                sum += num
                count += 1
        if count == 0:
            return 0
        return int(sum / count)